pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -35,20 +37,20 @@ const (
 )
 
 var (
-	healthCheckInterval = 300 * time.Second
+	healthCheckInterval = 3 * time.Minute
 )
 
 // Monitor defines OSD process monitoring
 type Monitor struct {
```

Comment:
Since we are renaming a bunch of things, I wonder if we could clarify the name of that struct too, perhaps something like `MonitorOSDHealth`?